core/domain/suggestion_services.py

```diff
@@ -821,6 +821,10 @@ def _has_translatable_exploration(suggestion, suggestion_exp_id_to_exp):
             content_id_exists = exploration.states[
                 suggestion.change.state_name].has_content_id(
                     suggestion.change.content_id)
+        if not content_id_exists:
+            logging.info(
+                'content_id does not exist for (suggestion ID, state_name, content_id): (%s, %s, %s)',
+                suggestion.id, suggestion.change.state_name, suggestion.change.content_id)
```

Comment:
Just a note for the future, suggestion object does not have an "id" property, it has a "suggestion_id" instead.